extension/background.js

```diff
@@ -21,13 +21,17 @@ async function getA11yScores() {
         for (var linkElement of linkElements) {
           // Adds a container with text score loading as the first child in each of the containers.
           let gContainer = linkElement.parentNode.parentNode.parentNode.parentNode;
-          if (!linkElement.hasAttribute('role') && gContainer.classList.contains('g') && linkElement.hasAttribute('data-ved') && !linkElement.getAttribute('href').startsWith('/search') && !gContainer.firstChild.innerHTML.startsWith('Score:')) {
-
-            let score = document.createElement("span");
-            gContainer.insertBefore(score, gContainer.firstChild);
+          let gContainerTwo = linkElement.parentNode.parentNode.parentNode.parentNode.parentNode.parentNode;
+          console.log(linkElement);
+          console.log(gContainer);
+          if (!linkElement.hasAttribute('role') && (gContainer.classList.contains('g') ||gContainerTwo.classList.contains('g') ||gContainerTwo.tagName.toLowerCase()=='li') && linkElement.hasAttribute('data-ved') && !linkElement.getAttribute('href').startsWith('/search')) {
```

Comment:
spaces between ||

Reply:
Good catch let me delete the console.logs and add the spaces